### PR TITLE
Implement a customer review sentiment classifier under prebuilt [resolves #192]

### LINKED ISSUE #192: Customer Satisfaction Prebuilt Model
Build a sentiment classification model on `pos`, `neg`, `neutral` targets of customer product reviews. [Candidate Dataset](https://www.kaggle.com/burhanbilenn/duygu-analizi-icin-urun-yorumlari)

Other datasets have 5 star rating as targets. They can be binned into sentiments. Or a different prebuilt implementation can be discussed.

### PR DESCRIPTION
**Pull request by dafajon**
**`bblock/doc.py`**:
- Implement a fix for returning zero vector instead of `nan` when the denominator/normalizing_factor is 0.

**`prebuilt/data`**:
- gzipped training data. Evaluation data is exclusive not included.

**`prebuilt/customer_sentiment.py`**
- Implement `empty_model`, `build` and `load` methods.
- Implement class mappings.
- Parameters for vectorizer and models are chosen in 3 fold cv. Optimizer code is exclusive not included.
- Save model with version. Use the version number for loading by adding `version` argument.

Update `prebuilt/README.md`with usage and evaluation accuracy.

### CODE REVIEW

**Comment by husnusensoy** [comment]
1. Built on 8K instances is not a serious model to be included into model
2. None of the code/data naming, layout standards is followed.

Thanks for your effort but kindly rejecting this PR
